src/ConstructiveSolidGeometry/SurfacePrimitives/EllipticalSurface.jl

```diff
@@ -190,3 +190,45 @@ end
 #         end
 #     end
 # end
+
+
+@inline _in_cyl_r(p::CartesianPoint, r::Real) = hypot(p.x, p.y) <= r
+@inline _in_cyl_r(p::CartesianPoint, r::Tuple{T,T}) where {T} =  r[1]<=hypot(p.x, p.y)<=r[2]
+@inline _in_cyl_r(p::CylindricalPoint, r::Union{Real, Tuple{T,T}}) where {T} =  _in_cyl_r(CartesianPoint(p), r)
+
+@inline _left_radial_interval(r::Real) = 0
+@inline _left_radial_interval(r::Tuple{T,T}) where {T} = r[1]
+
+@inline _right_radial_interval(r::Real) = r
+@inline _right_radial_interval(r::Tuple{T,T}) where {T} = r[2]
+
+get_r_limits(a::EllipticalSurface{T, <:Union{T, <:Any, Nothing}, <:Any}) where {T} =
+    (_left_radial_interval(a.r), _right_radial_interval(a.r))
```

Comment:
This seems to do exactly what `_radial_endpoints` in `src/ConstructiveSolidGeometry/Intervals.jl` does.